### DISCUSSION TITLE
Is there a command line switch to disable the generation of timestamps?

### DISCUSSION THREAD

**Discussion by Robert-K-S** [openai/whisper#1320] [Q&A] [answered]
Or, if not, can I mention this as a feature request?

**Comment by jongwook** [comment, marked as answer]
If you're referring to the `<|notimestamps|>` token, the command line interface (and the `transcribe()` function) does not have an option to do so because it uses the timestamps to determine the next 30-second window to run transcription.

If you're only interested in short audio segments (shorter than 30s), you can use `decode()` with `without_timestamps=True`:

https://github.com/openai/whisper/blob/248b6cb124225dd263bb9bd32d060b6517e067f8/whisper/decoding.py#L107-L108
